atmsimulator/login.py

```diff
@@ -0,0 +1,30 @@
+import inquirer
+import requests
+import sys
+from simple_chalk import chalk
+from pyfiglet import figlet_format
+
+from .globals import account

```

Comment:
Please do check if this assignment of account actually updates global state since files would be utilising the same `globals.py` file.